setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup

```

Comment:
I used this file to install for development using `pip install -e .`. I did not know how to do this otherwise. Let me know if I should remove it again?

Reply:
You shouldnt need this file to install, unless you're using an old version of pip it should support pyproject.toml. 
If you think we should add a setup.py to support older versions of pip, it should be auto generated from the pyproject.toml file using `dephell deps convert` so they are consistent. But I would argue if someone wants to contribute its not to much to ask for them to update their pip first.